ui/src/components/analytics.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react'
+import { useCookieConsent } from 'utils/cookieConsent/cookieConsentContext'
+
+const GTM_ID = 'G-EX26RXX0YT'
+
+export const Analytics = () => {
```

Comment:
In #351 we said to use a React lib for this, however [the most popular one](https://github.com/alinemorelli/react-gtm) for Google Tag Manager did not work out of the box and after a closer look it seemed pretty outdated.

I suggest we use this custom implementation, at least until we have more advanced analytics. It's pretty much same as before, except we load and unload the scripts depending on cookie consent status.